github.js

```diff
@@ -178,7 +178,7 @@ GitHub.Diff = class {
       const file_name = $(file_header).attr('data-path');
       const file_anchor = $(file_header).attr('data-anchor');
 
-      if (context.match(/^[ -+]?\$ANSIBLE_VAULT;.+AES256$/) && !$('#' + file_anchor + '-decryptbtn').length) {
+      if (context.match(/^[ \-+]?\$ANSIBLE_VAULT;.+AES256$/) && !$('#' + file_anchor + '-decryptbtn').length) {
```

Comment:
ah yep.. missed that too. The usual method of including a dash in a character set is to make it the first or last character in the set so it's not treated as a range, rather than having to escape it - so `/^[ +-]?...`